src/relax/transform/lift_transform_params.cc

```diff
@@ -144,6 +144,8 @@ class TransformParamsFuncBuilder : public ExprMutator {
  */
 class LiftTransformParamsPlanner : public ExprVisitor {
  public:
+  explicit LiftTransformParamsPlanner(IRModule mod) : mod_(mod) {}
+
```

Comment:
Just to confirm, are we using the IRModule somewhere? I seems to find no use in both LiftTransformParamsPlanner and TransformParamsLifter.

Reply:
sorry it's some code used for attr. since we have changed to op, we can delete them